### DISCUSSION TITLE
Way to specify version of album

### DISCUSSION THREAD

**Discussion by MadDucks** [RandomNinjaAtk/docker-lidarr-extended#124] [Q&A] [answered]
Is there a way to specify the version of the album that you want?  I have a Deluxe version that I want grabbed.

**Comment by RandomNinjaAtk** [comment, marked as answer]
I don't believe there is any method to specify. However, from what I recall, it is designed to go for the biggest release first, so it's preference should always be the Deluxe edition, or the one with the most tracks.
